libraries/wallet/wallet_api_impl.cpp

```diff
@@ -56,9 +56,15 @@ namespace graphene { namespace wallet { namespace detail {
         _remote_api(rapi),
         _remote_db(rapi->database()),
         _remote_net_broadcast(rapi->network_broadcast()),
-        _remote_hist(rapi->history()),
-        _custom_operations(rapi->custom())
+        _remote_hist(rapi->history())
    {
+      try {
+         _custom_operations = rapi->custom();
+      }
+      catch(const fc::exception& e)
+      {
+         wlog("Custom operations plugin is not loaded.");
```

Comment:
Better: `Custom operations API is not active on server.`
What happens in this case if `get_account_storage` is invoked?

Reply:
> Better: Custom operations API is not active on server.
Changed.
>  What happens in this case if get_account_storage is invoked?
Crash, added `try` to call, thanks :)

Both changes at https://github.com/bitshares/bitshares-core/pull/2033/commits/f6738ec1108d36a9700641eb31a4c0274302f00f

Note: No need to add `try` at `account_store_map` as it does not use the API. custom operations can be created without the plugin or the api being enabled, not possible to read without them.